packages/expo-application/src/Application.ts

```diff
@@ -126,7 +126,7 @@ export async function getIosIdForVendorAsync(): Promise<string | null> {
 // @needsAudit
 /**
  * Gets the iOS application release type.
- * @return Returns a promise which fulfills with an [`ApplicationReleaseType`](#applicationreleasetype).
+ * @return A `Promise` which fulfills with an [`ApplicationReleaseType`](#applicationreleasetype).
```

Comment:
Since we use `@return` annotation in many API docs, I've removed the redundancy to maintain consistency with the rest of the docs.